File: Day-14/Pattern18.py
n = int(input())
def PatternEighteen(n):
    i = 1
    while (i <=n):
        j = 1
        while (j <= i ):
            print(chr(65 + n + j - i -1), end = '')
            j +=1
        i += 1
        print(end='\n')
    
PatternEighteen(n)

File: Day-15/Pattern18.js
function PatternEighteen(n) {
  let i = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= i) {
      str += String.fromCharCode(65 + n + j - i - 1);
      j++;
    }
    console.log(str);
    i++;
  }
}

PatternEighteen(4);
